docs/en/integrations/language-clients/js.md

```diff
@@ -47,17 +57,26 @@ npm i @clickhouse/client-web
 
 ## Compatibility with ClickHouse
 
-| Client version | ClickHouse   |
-|----------------|--------------|
-| 0.2.9          | 22.8 - 23.12 |
+| Client version | ClickHouse  |
+|----------------|-------------|
+| 0.3.0          | 22.8 - 24.2 |
```

Comment:
Do we run tests against `22.8`? Why? `23.3` is the latest supported https://github.com/ClickHouse/ClickHouse/blob/master/SECURITY.md

Reply:
Update this part; thanks for pointing it out.